be/src/exprs/string_functions.cpp

```diff
@@ -3866,6 +3866,27 @@ StatusOr<ColumnPtr> StringFunctions::money_format_double(FunctionContext* contex
     return result.build(ColumnHelper::is_all_const(columns));
 }
 
+StatusOr<ColumnPtr> StringFunctions::format_double(FunctionContext* context, const starrocks::Columns& columns) {
+    auto data_viewer = ColumnViewer<TYPE_DOUBLE>(columns[0]);
+    auto decimal_viewer = ColumnViewer<TYPE_INT>(columns[1]);
```

Comment:
three suggestions: 
1. to rename "decimal_xxx" to  "scale_xxxx" , decimal has already be used as a type;
2. return const null column fast if any parameter is null;
3. 2nd parameter would be constant in most of scenarios,  please adopt constant column optimization, reference to substr function implementation.